content/doc/book/security/managing-security.adoc

```diff
@@ -21,6 +21,25 @@ Pages to mark as deprecated by this document:
 https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/jenkins/security/s2m/MasterKillSwitchConfiguration/help-masterToagentAccessControl.html#L2
 /content/redirect/security-144
 
+[NOTE]
+.Enable Security
```

Comment:
Time to delete? It's been more than 18 months since the last release for which this matters.

Reply:
That makes sense to me. Do you want to wait for others to weigh in here?

Reply:
+1 for removal